lib/YuiRestClient.pm

```diff
@@ -25,8 +25,8 @@ my $host;
 
 sub get_app {
     my (%args) = @_;
-    die "No environment for libyui REST API has been configured.",
-      " Please schedule test module `setup_libyui.pm` in previous steps." unless YuiRestClient::is_libyui_rest_api();
+    die "This module is using LibYUI REST API but there is no such configuration in place. ",
```

Comment:
the previous message was correct and more clear in this specific point when reached after setup_libyui (although I would prefer not to mention specific test modules names, just setup libyui, for example, in case name would change). 

I think the problem is that we need to control the error in setup_libyui.pm, there is not point to get the app in `run` if the variable is not set, there is where we can provide a specific message "do you forget to set this variable? blablab". wdyt?

Reply:
I agree, I thought that yesterday on the discussion we had decided to keep things simple and just alter the message, my bad.